README.md

```diff
@@ -175,6 +211,50 @@ The post-processor requires additional permissions to work. Below is the differe
 }
 ```
 
+Add this additional IAM Policy to the IAM Role if you set `resolve_aliases` to `true`
+
+```json
+{
+    "Statement": [
+        {
+            "Action": [
+                "ssm:GetParameters"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
+        }
+    ],
+    "Version": "2012-10-17"
+}
+```
+
+As a best practice you can restrict the permission to only a specific subgroup of parameters.
+
+```json
+{
+    "Statement": [
+        {
+            "Action": [
+                "ssm:GetParameters"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:ssm:${region}:${accountId}:parameter/company-amis/*"
+        }
+    ],
+    "Version": "2012-10-17"
+}
+```
+
+NOTE: Replace `${region}` and `${accountId}` with the correct values.
+
+In the previous example. We give access to all parameters whose name begin with `/company-amis/`. For example if you had the following parameters:
+
+* /company-amis/wordpress/windows
+* /company-amis/wordpress/linux
+* /cool-service/db-password
+
+This permission would give access to `/company-amis/wordpress/windows` and `/company-amis/jomla/linux` parameters. But not to `/cool-service/db-password` parameter.
+
```

Comment:
There may be no need to go into this much detail about IAM roles. I think adding `ssm:GetParameters` to the above IAM example will be sufficient.

Reply:
Got it. Will update

Reply:
Removed, the extra detail.